docs/topic/infrastructure/storage-layer.md

```diff
@@ -128,7 +128,7 @@ jupyterhub:
           # Mounted without readonly attribute here,
           # so we can chown it appropriately
           - name: home
-            mountPath: /home/jovyan/shared-readwrite
+            mountPath: /home/jovyan/shared
             subPath: _shared
```

Comment:
The `mountPath` is irrelevant for the `chown` operation as the `_subpath` is influenced the same no matter what. By adjusting the `mountPath` to `shared` we avoid creating an empty directory `shared-readwrite` owned by root in the user's home directory and instead get a `shared` empty directory. But, this isn't a problem as it will be hidden by a mount later anyhow no matter if its an admin user or not.